src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -185,6 +189,30 @@ public function isInitializable(string $class, string $property, array $context
         return false;
     }
 
+    private function extractFromDeclaredType(string $class, string $property)
+    {
+        // to be removed as soon as Symfony bumps the minimum PHP Version to 7.4
+        if (version_compare(PHP_VERSION, '7.4.0-dev', '<')) {
```

Comment:
this should use `\PHP_VERSION_ID < 70400` instead. This matches our conventions, which are set this way because it allows to have the condition resolved at compile-time (and so OPCache can optimize the code)

Reply:
Hi @stof, fixing the version to `70400` will not match the dev release (e.g 7.4.0-dev) and then this PR will not work `7.4snapshot` image of travisci. Any idea to solve this issue?